### ISSUE TITLE
Traceback Error after running "slither . "

### ISSUE THREAD

**Issue by leonardishere** [crytic/slither#807]
Installed slither with pip3, using hardhat 2.0.11, @openzeppelin/contracts 4.0.0, fresh npm install, fresh hardhat compile, my unit tests pass, but slither throws errors. It looks like it's having problems finding the imports.
```
$ slither --version
0.7.0

$ python --version
Python 3.7.7

$ solc --version
solc, the solidity compiler commandline interface
Version: 0.8.0+commit.c7dfd78e.Linux.g++
```
```
$ slither .
'npx hardhat compile' running
Nothing to compile
Creating Typechain artifacts in directory typechain for target ethers-v5
Successfully generated Typechain artifacts!

Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/__main__.py", line 712, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/__main__.py", line 82, in process_all
    ) = process_single(compilation, args, detector_classes, printer_classes)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/__main__.py", line 65, in process_single
    slither = Slither(target, ast_format=ast, **vars(args))
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/slither.py", line 102, in __init__
    self._parser.parse_contracts()
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/solc_parsing/slitherSolc.py", line 380, in parse_contracts
    self._analyze_second_part(contracts_to_be_analyzed, libraries)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/solc_parsing/slitherSolc.py", line 459, in _analyze_second_part
    self._analyze_struct_events(contract)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/solc_parsing/slitherSolc.py", line 510, in _analyze_struct_events
    contract.analyze_using_for()
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/solc_parsing/declarations/contract.py", line 530, in analyze_using_for
    lib_name = parse_type(using_for["libraryName"], self)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/slither/solc_parsing/solidity_types/type_parsing.py", line 356, in parse_type
    raise ParsingError("Type name not found " + str(t))
slither.solc_parsing.exceptions.ParsingError: Type name not found {'id': 584, 'name': 'Address', 'nodeType': 'IdentifierPath', 'referencedDeclaration': 1098, 'src': '608:7:2'}
ERROR:root:Error:
ERROR:root:Type name not found {'id': 584, 'name': 'Address', 'nodeType': 'IdentifierPath', 'referencedDeclaration': 1098, 'src': '608:7:2'}
ERROR:root:Please report an issue to https://github.com/crytic/slither/issues
```
```
$ slither contracts/File.sol
Compilation warnings/errors on contracts/File.sol:
Error: Source "@openzeppelin/contracts/utils/math/Math.sol" not found: File not found.
 --> contracts/File.sol:4:1:
  |
4 | import "@openzeppelin/contracts/utils/math/Math.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Error: Source "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol" not found: File not found.
 --> contracts/File.sol:5:1:
  |
5 | import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Error: Source "@openzeppelin/contracts/utils/Address.sol" not found: File not found.
 --> contracts/File2.sol:4:1:
  |
4 | import "@openzeppelin/contracts/utils/Address.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Error: Source "@openzeppelin/contracts/token/ERC20/ERC20.sol" not found: File not found.
 --> contracts/File2.sol:5:1:
  |
5 | import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Error: Source "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol" not found: File not found.
 --> contracts/File2.sol:6:1:
  |
6 | import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by montyly** [comment]
Hi @leonardishere, thank you for reporting this.

Can you share the codebase where this happens?

**Comment by leonardishere** [comment]
Hi @montyly I created a minimal example at https://github.com/leonardishere/SlitherExample

**Comment by nichanank** [comment]
Running into the same issue. At first I thought it was due to the external imports from openzeppelin but even copying these files and having it in the `contracts/` folder I'm running into the same error
![image](https://user-images.githubusercontent.com/15038946/113485245-c8572a80-94d6-11eb-8bf8-c70777fbca95.png)

**Comment by nichanank** [comment]
Upon further inspection it looks like this issue appears for all custom type names from OpenZeppelin. I am running into the same error for types like `EnumerableSet`, `SafeERC20`, `Counters` imported from OZ. The error remains even after bringing these contracts into the local directory (not relying on external imports)

**Comment by montyly** [comment]
Hi @nichanank. Thanks for the detailed information. I am going to take a look at this bug this week

**Comment by montyly** [comment]
Hi @leonardishere / @nichanank. We fixed the issue in #817, the fix will be available in the upcoming release (https://github.com/crytic/slither/issues/829)
